frontend/src/Components/UserProfileKarma.tsx

```diff
@@ -60,23 +70,73 @@ export const UserProfileKarma = (props: UserProfileKarmaProps) => {
+        <><span className={'i i-no-poop'}></span>Нет права ставить плюсы и минусы постам и комментариям. Голоса в карму другим людям отменены.</>,
+
+        restrictionsResult.restrictedToPostId === true &&
+        <><span className={'i i-dead'}></span>Права максимально ограничены, есть возможность создать свой последний пост.</>,
+
+        Number.isFinite(restrictionsResult.restrictedToPostId) &&
+        <>
+            <span className={'i i-dead'}></span>
+            <div>Права максимально ограничены, есть возможность писать только в своем
+            <PostLink post={{
+                id: restrictionsResult.restrictedToPostId as number,
+                site: 'main'
+            }}> последнем посте</PostLink>.
+            </div>
+        </>,
+    ].filter(Boolean);
+
+    if (restrictions && restrictions.length === 0) {
+        restrictions.push(<><span className={'i i-thumbs-up'}></span>Ура! Права не ограничены!</>);
+    }
+
     return (
-        !karmaResult || !restrictionsResult ?
-            <div>Загрузка...</div>
-            : <>
-                <div>
-                    {restrictionsResult.restrictedToPostId === true &&
-                        <span className={styles.restricted}>Может создать один последний пост</span>}
-                    { Number.isFinite(restrictionsResult.restrictedToPostId) &&
-                        <span className={styles.restricted}>
-                            Может комментировать только&nbsp;
-                            <PostLink post={{ id : restrictionsResult.restrictedToPostId as number, site: 'main'}}>в посте</PostLink>
-                        </span>}
-                    { restrictionsResult.commentSlowModeWaitSec > 0 && <span className={styles.restricted}>Временно не может комментировать</span>}
-                    { restrictionsResult.postSlowModeWaitSec > 0 && <span className={styles.restricted}>Временно не может создавать посты</span>}
-                    { !restrictionsResult.canVote && <span className={styles.restricted}>Не может голосовать</span>}
-                    { !restrictionsResult.canInvite && <span className={styles.restricted}>Не может приглашать</span>}
-                </div>
+        <>
+            {isOwnProfile && <div className={styles.info}>
+                <div className={styles.beta}>BETA</div>
+                <p>Мы продолжаем работать над саморегуляцией.</p>
+                <p>Это самая первая версия, работающая по механизму,
+                <PostLink post={{id: 781, site: 'dev'}}> описанному тут</PostLink>.</p>
+                <p>Формула саморегуляции зависит от двух вещей — оценок другими людьми вас и вашего контента (постов и комментариев).
+                    Если коротко, ведите себя по-человечески, производите хороший контент, и все будет хорошо.</p>
+            </div>}
+
+            {!restrictionsResult && <div>Загрузка...</div>}
+            {!!restrictions?.length && <div>{restrictions.map((r, idx) =>
```

Comment:
I think `!!` is not needed here unless I'm missing something

Reply:
React renders `{0}` as "0",  while `{false}` is not rendered.  `0 && something` evaluates to `0`.